hsds/servicenode_lib.py

```diff
@@ -1006,3 +1011,116 @@ async def deleteObj(app, obj_id, bucket=None):
     meta_cache = app["meta_cache"]
     if obj_id in meta_cache:
         del meta_cache[obj_id]  # remove from cache
+
+
+async def createGroup(app, root_id=None, creation_props=None, bucket=None):
```

Comment:
Since the parent group that will contain the new group doesn't have to be the root group, `root_id` should be renamed to `parent_id`

Reply:
This method is for anonymous group so parent I’d wouldn’t apply.  The root I’d basically determines which domain the group will belong to

Reply:
ID not Id